about.css

```diff
@@ -0,0 +1,466 @@
+* {
+  font-family: "Lato", sans-serif;
+}
+
+.home-page {
+  background-image: url(./assets/About/hero-mobilebg.svg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  padding: 12px 0 55px 0;
+}
+
+.toolbar {
+  display: flex;
+  justify-content: start;
+  gap: 21%;
+  padding-left: 20px;
+}
+
+.hero-intro > img {
+  width: 90%;
+  padding-bottom: 20px;
+}
+
+.about_logo img {
+  width: 100%;
+}
+
+body > section > div > a:nth-child(1) > img {
+  margin: 25% auto;
+}
+
+.home-content {
+  background: #fff;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 25px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: -0.06em;
+  color: #000;
+  padding: 24px 10px;
+  width: 100%;
+}
+
+.home-date {
+  font-style: normal;
+  font-weight: 800;
+  font-size: 12px;
+  line-height: 25px;
+  text-align: center;
+  letter-spacing: 0.02em;
+  text-decoration-line: underline;
+  color: #5f5f5f;
+}
+
+.at {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 25px;
+  text-align: center;
+  color: #7a7a7a;
+  margin: 0 44px;
+}
+
+.info {
+  text-align: center;
+}
+
+.info h2 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 28px;
+  letter-spacing: -0.02em;
+  color: #2c2c2c;
+  margin: 20px 74px 5px;
+}
+
+.info p {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 21px;
+  letter-spacing: -0.03em;
+  color: #838383;
+  margin: 20px 10px;
+}
+
+.about_logo {
+  margin-bottom: 20px;
+}
+
+.abt-img > img:nth-child(2) {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.partner {
+  text-align: center;
+  background-color: #414246;
+  color: #fff;
+}
+
+.partner h2 {
+  margin: 0;
+}
+
+.img-logo {
+  padding-bottom: 25px;
+}
+
+.img-logo ul {
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0;
+  justify-content: space-around;
+  align-items: center;
+  margin: 0 20px;
+}
+
+.img-logo li {
+  list-style: none;
+  padding: 10px;
+}
+
+.footer {
+  display: flex;
+  margin: 0 18px;
+  gap: 15%;
+  padding: 20px;
+}
```

Comment:
- [ ] Please as mentioned by the previous reviewer kindly set the footer's margin to 0, to avoid the gap around. Please kindly fix that 😉 😉 

Your design

<img width="949" alt="Screenshot 2023-03-23 at 14 46 38" src="https://user-images.githubusercontent.com/74501523/227198607-96992a27-43aa-4067-97ce-7f25aaf85214.png">

Original design

<img width="873" alt="Screenshot 2023-03-23 at 14 48 18" src="https://user-images.githubusercontent.com/74501523/227198652-f36305e1-e7bd-4a44-99f9-db931d4d3221.png">